patterns/sso-iam-identity-center/cam/README.md

```diff
@@ -0,0 +1,112 @@
+# IAM Identity Center Single Sign-On for Amazon EKS Cluster with Cluster Access Manager
```

Comment:
If we drop `aws-auth` pattern, we can tidy up the name/descriptions by going back to just simply `sso-iam-identity-center` and `IAM Identity Center Single Sign-On for Amazon EKS Cluster`

Reply:
Agree to to this movement after drop the `aws-auth`